docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -65,7 +65,8 @@ execution on a Trino cluster:
 * - `retry-policy`
   - Configures what is retried in the event of failure, either `QUERY` to retry
     the whole query, or `TASK` to retry tasks individually if they fail. See
-    [retry policy](fte-retry-policy) for more information.
+    [retry policy](fte-retry-policy) for more information. The equivalent
```

Comment:
Content change one

Reply:
Not confident - using session property is not users should do most of the time. This to large extent implies mixing fte and non-fte workloads on the same cluster - which is not tested configuretion and has known limitations, as discussed recently on slack.

Unless this is about NONE/QUERY - those are fine.

Reply:
We have to add that info into the docs. We need to be very explicit .. users want one cluster to mix it and it just doesnt work. I will update this PR ..